poster/styles/styles.scss

```diff
@@ -11,480 +7,451 @@ sans-serif;
-    grid-column: 4/5;
-    grid-row: 14/20;
-}
+    &thirtytwo {
+        margin-left: 4vw;
+    }
 
-.titlefifthletter {
-    font-family: $thick-display;
-    font-size: 28vw;
-    grid-column: 5/6;
-    grid-row: 1/7;
-    margin-left: -6vw;
-    transform: rotate(-30deg);
-}
+    &thirtytwosecond {
+        margin-left: 2vw;
+    }
 
-.titlesixthletter {
-    font-family: $thin-display;
-    grid-column: 6/7;
-    grid-row: 14/20;
-}
+    &thirtythree {
+        margin-left: 8vw;
+    }
 
-.titleseventhletter {
-    font-family: $thin-display;
-    grid-column: 7/8;
-    grid-row: 10/16;
-}
+    &thirtyfour {
+        display: inline-block;
+        margin-top: 2vw;
+        margin-left: 18vw;
+    }
 
-/*title location within outer wrapper*/
+    &thirtyfive {
+        margin-left: 16vw;
+    }
 
-.tophalf {
-    grid-row: 1/2;
-    grid-column: span;
-}
+    &thirtysix {
+        margin-left: 13.6vw;
+    }
 
-ul {
-    list-style-type: none;
-}
+    &thirtyseven {
+        margin-left: 10.8vw;
+    }
 
-@media (max-width:850px) and (min-width:0px) {
+    &thirtyeight {
+        margin-left: 8.4vw;
+    }
 
-    /*body text font size and weight CHANGE*/
-    $font-sizemediaqone: 3vw;
+    &thirtynine {
+        margin-left: 6vw;
+    }
 
-    /*redefined margins for body*/
-    .body-lineone {
-        padding-top: 1vw;
+    &forty {
+        margin-left: 4vw;
     }
 
-    .body-linefour {
-        margin-left: 49vw;
+    &fortyone {
+        margin-left: 2vw;
     }
 
-    .body-linefive {
-        margin-top: -3vw;
-        margin-left: 39vw;
+    &fortythree {
+        display: block;
+        margin-top: -4.3vw;
+        margin-left: 30vw;
     }
 
-    .body-linetwelve {
-        margin-bottom: -3vw;
+    &fortyfour {
+        margin-left: 28vw;
     }
 
-    .body-linetwentytwo {
-        margin-top: 0;
-        margin-left: 29vw;
-        margin-bottom: 4vw;
+    &fortyfive {
+        margin-left: 25vw;
     }
 
-    .body-linetwentyfive {
-        margin-left: 6vw;
+    &fortysix {
+        margin-left: 21vw;
     }
 
-    .body-linethirty {
-        margin-bottom: -3vw;
+    &fortyseven {
+        margin-left: 18vw;
     }
 
-    .body-linethirtyone {
-        margin-left: 6vw;
+    &fortyeight {
+        margin-left: 15vw;
     }
 
-    .body-linethirtytwo {
-        margin-left: 2vw;
+    &fortynine {
+        margin-left: 11vw;
     }
 
-    .body-linethirtytwosecond {
-        margin-left: 0vw;
+    &fifty {
+        margin-left: 8vw;
     }
 
-    .body-linefortythree {
-        margin-top: -7.5vw;
-        margin-left: 50vw;
-        margin-bottom: -4vw;
+    &fiftyone {
+        margin-left: 4vw;
     }
 
-    .body-linefortyfour {
-        margin-left: 47vw;
+    &fiftytwo {
+        margin-left: 8vw;
     }
 
-    .body-linefortyfive {
-        margin-left: 43vw;
+    /*display block to move up like in poster*/
+
+    &fiftythree {
+        font-size: .7vw !important; // needed to override default paragraph font-size
```

Comment:
The impotant flag was needed here to override the font-size of the default `p` element.